tango_simlib/tests/config_files/database2.fgo

```diff
@@ -1 +1 @@
operty\t 0.000\t 0.000\t 0.000\t0", "DbPutDeviceProperty\t 1.649\t 0.913\t 5.195\t7", "DbUnExportServer\t 0.000\t 0.000\t 0.000\t0"], "polling": 0, "time": 1521724934.786855, "alarms": {"delta_t": "Not specified", "max_alarm": "Not specified", "min_warning": "Not specified", "extensions": "[]", "delta_val": "Not specified", "min_alarm": "Not specified", "max_warning": "Not specified"}, "model": "monctl:10000/sys/database/2/Timing_info"}, "StoredProcedureRelease": {"color": "Lime", "display_unit": "No display unit", "writable": "READ", "standard_unit": "No standard unit", "quality": "ATTR_VALID", "unit": "", "string": "release 1.13", "label": "StoredProcedureRelease", "min_alarm": "Not specified", "events": {"per_event": {"extensions": "[]", "period": "1000"}, "ch_event": {"rel_change": "Not specified", "extensions": "[]", "abs_change": "Not specified"}, "arch_event": {"archive_period": "Not specified", "extensions": "[]", "archive_rel_change": "Not specified", "archive_abs_change": "Not specified"}}, "description": "", "data_type": "DevString", "format": "%s", "max_alarm": "Not specified", "device": "sys/database/2", "name": "StoredProcedureRelease", "database": "monctl:10000", "data_format": "SCALAR", "value": "release 1.13", "polling": 0, "time": 1521724934.722425, "alarms": {"delta_t": "Not specified", "max_alarm": "Not specified", "min_warning": "Not specified", "extensions": "[]", "delta_val": "Not specified", "min_alarm": "Not specified", "max_warning": "Not specified"}, "model": "monctl:10000/sys/database/2/StoredProcedureRelease"}, "Timing_minimum": {"color": "Lime", "display_unit": "No display unit", "writable": "READ", "standard_unit": "No standard unit", "quality": "ATTR_VALID", "unit": "", "string": "  0.29,  0.32,  0.00,  0.00,  9.23,  0.00,  0.00,  0.00,  0.39,  0.00,  0.00,  0.00,  0.00,  0.00,  0.00,  0.00,  0.62,  0.24,  0.16,  0.00,  1.00,  0.19,  3.37,  0.00,  0.00,  0.00,  0.91,  0.00", "label": "Timing_minimum", "min_alarm": "Not specified", "events": {"per_event": {"extensions": "[]", "period": "1000"}, "ch_event": {"rel_change": "Not specified", "extensions": "[]", "abs_change": "Not specified"}, "arch_event": {"archive_period": "Not specified", "extensions": "[]", "archive_rel_change": "Not specified", "archive_abs_change": "Not specified"}}, "description": "", "data_type": "DevDouble", "format": "%6.2f", "max_alarm": "Not specified", "device": "sys/database/2", "name": "Timing_minimum", "database": "monctl:10000", "data_format": "SPECTRUM", "value": [0.29, 0.32, 0.0, 0.0, 9.23, 0.0, 0.0, 0.0, 0.39, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.62, 0.24, 0.16, 0.0, 1.0, 0.19, 3.37, 0.0, 0.0, 0.0, 0.91, 0.0], "polling": 0, "time": 1521724934.740163, "alarms": {"delta_t": "Not specified", "max_alarm": "Not specified", "min_warning": "Not specified", "extensions": "[]", "delta_val": "Not specified", "min_alarm": "Not specified", "max_warning": "Not specified"}, "model": "monctl:10000/sys/database/2/Timing_minimum"}}, "properties": {}}
```

Comment:
Why were the SPECTRUM (1D) attributes removed?  I would expect them to work.  Do you know if the DSH LMC has any spectrum attributes?

Reply:
It gives a key error when the launcher is run because it's missing max_dim_x

Reply:
As discussed, we need to do some work in the fandago parser to infer values for the missing keys, since the fandango export file doesn't include them.  That can be done in a subsequent PR.